LN-Zap/zap-desktop
# fix(ui): ensure copy/paste works in amount fields

## Description:

Fix issue that was preventing copy/paste from working in the amount fields (eg on the request form)

## Motivation and Context:

Fix #1389

## How Has This Been Tested?

- Verify that copy/paste works in amount fields
- Verify that amount fields continue to work as expected

## Types of changes:

Bug fix

## Checklist:
